### ISSUE TITLE
about pre-train code 

### ISSUE THREAD

**Issue by tian961214** [lyxok1/STM-Training#5]
First of all, thank you for your training code, but the accuracy obtained in this way is about 69.4. Do you have a pre-train code? If it is convenient for you, please release it.

**Comment by haochenheheda** [comment]
> First of all, thank you for your training code, but the accuracy obtained in this way is about 69.4. Do you have a pre-train code? If it is convenient for you, please release it.

Hi, tian. I have shared my reproduced pretrain code [here](https://github.com/haochenheheda/Training-Code-of-STM)

**Comment by longmalongma** [comment]
> First of all, thank you for your training code, but the accuracy obtained in this way is about 69.4. Do you have a pre-train code? If it is convenient for you, please release it.

me too, I had the same problem!
